pkg/machines/components/vm/bootOrderModal.jsx

```diff
@@ -0,0 +1,319 @@
+        addOptional(additionalInfo, device.device.source.dev, _("Device"));
+        addOptional(additionalInfo, device.device.source.protocol, _("Protocol"));
+        addOptional(additionalInfo, device.device.source.pool, _("Pool"));
+        addOptional(additionalInfo, device.device.source.volume, _("Volume"));
+        addOptional(additionalInfo, device.device.source.host.name, _("Host"));
+        addOptional(additionalInfo, device.device.source.host.port, _("Port"));
+        break;
+    }
+    case "network": {
+        heading = rephraseUI("bootableDisk", "network");
+        addOptional(additionalInfo, device.device.mac, _("Mac"));
+        break;
+    }
+    case "redirdev": {
+        heading = rephraseUI("bootableDisk", "redirdev");
+        addOptional(additionalInfo, device.device.type, _("Type"));
+        addOptional(additionalInfo, device.device.bus, _("Bus"));
+        addOptional(additionalInfo, device.device.address.port, _("Port"));
+        break;
+    }
+    case "hostdev": {
+        heading = rephraseUI("bootableDisk", "hostdev");
+        const nodeDev = findHostNodeDevice(device.device, nodeDevices);
+        if (nodeDev) {
+            switch (device.device.type) {
+            case "usb": {
+                addOptional(additionalInfo, device.device.type, _("Type"));
+                addOptional(additionalInfo, nodeDev.capability.vendor._value, _("Vendor"));
+                addOptional(additionalInfo, nodeDev.capability.product._value, _("Product"));
+                break;
+            }
+            case "pci": {
+                addOptional(additionalInfo, device.device.type, _("Type"));
+                addOptional(additionalInfo, nodeDev.capability.vendor._value, _("Vendor"));
+                addOptional(additionalInfo, nodeDev.capability.product._value, _("Product"));
+                break;
+            }
+            case "scsi": {
+                addOptional(additionalInfo, device.device.type, _("Type"));
+                addOptional(additionalInfo, device.device.source.address.bus, _("Bus"));
+                addOptional(additionalInfo, device.device.source.address.target, _("Target"));
+                addOptional(additionalInfo, device.device.source.address.unit, _("Unit"));
+                break;
+            }
+            case "scsi_host": {
+                addOptional(additionalInfo, device.device.type, _("Type"));
+                addOptional(additionalInfo, device.device.source.protocol, _("Protocol"));
+                addOptional(additionalInfo, device.device.source.wwpn, _("WWPN"));
+                break;
+            }
+            case "mdev": {
+                addOptional(additionalInfo, device.device.type, _("Type"));
+                addOptional(additionalInfo, nodeDev.capability.type.id, _("Type ID"));
+                break;
+            }
+            }
+            break;
+        }
+    }
+    }
+
+    let upArrow;
```

Comment:
These are constants.

Reply:
I don't know if I get what do you mean. I think declaring const variables without assignment would return error.

Reply:
How you do it now you can't use constant because you have the assignment inside the if statement. You can use if you make in one liner with conditional assignment. Feel free to leave it as is. Didn't notice before that the assignment is in conditional.

Reply:
That would be quite long one liner. I feel like this it looks nicer.